BUILD.md

```diff
@@ -249,4 +249,18 @@ Overall coverage rate:
 
 The report will be generated in HTML form to alow visualization of each line of code coverage for each C++ file in the project. The entry HTML page to the report is located at the C++ build folder: `cpp/build/coverage/index.html`
 
-In VSCode, line by line coverage can be visualized using the extension [gcov-viewer](https://marketplace.visualstudio.com/items?itemName=JacquesLucke.gcov-viewer)
\ No newline at end of file
+In VSCode, line by line coverage can be visualized using the extension [gcov-viewer](https://marketplace.visualstudio.com/items?itemName=JacquesLucke.gcov-viewer)
+
+
+## Fuzzing
+
+Go code can be easily fuzzed: fuzzing attempts to find bugs by mutating a test input data set and measuring test coverage. This complementary technique aims stability issues.
+
+### Fuzzing Evm.StepN interface
+
+The ct Evm.StepN interface is used to evaluate one N instructions in different EVM implementations. There are 2 Fuzzer tests against this interface:
+
+- Crash test: FuzzLfvm will execute instructions one at the time looking for a panic. ```make fuzz-lfvm```
+- Differential test: FuzzDifferentialLfvmVsGeth ```make fuzz-lfvm-diff```
```

Comment:
maybe one phrase about this command as well ?

Reply:
It should be fixed now. Thanks for checking.